scripts/setup-adapters.sh

```diff
@@ -147,8 +147,12 @@ function install_hdfs_deps {
     sed -i "/FIND_PACKAGE(GoogleTest REQUIRED)/d" ./CMakeLists.txt
     sed -i "s/dumpversion/dumpfullversion/" ./CMake/Platform.cmake
     # Dependencies for Hadoop testing
-    wget_and_untar https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz hadoop
+    wget_and_untar https://archive.apache.org/dist/hadoop/common/hadoop-3.3.0/hadoop-3.3.0.tar.gz hadoop
     cp -a hadoop /usr/local/
+    export CLASSPATH=`/usr/local/hadoop/bin/hdfs classpath --glob`
+    
+    yum install -y java-1.8.0-openjdk-devel
+    export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
```

Comment:
java is already installed in the adapters docker image: https://github.com/facebookincubator/velox/blob/main/scripts/adapters.dockerfile#L36 so this can be removed. The class path var also needs to be set in the dockerfile with `ENV` as the `export` will not persist after `RUN`

Reply:
@assignUser Apologies for the delayed response. I have relocated the export to https://github.com/facebookincubator/velox/blob/main/scripts/adapters.dockerfile#L36. However, I am unable to find the Java installation code within the adapters Dockerfile. Consequently, I have retained the command to install Java.

Reply:
It's installed via mamba: https://github.com/facebookincubator/velox/blob/main/scripts/adapters.dockerfile#L33C1-L33C36

Reply:
Ah and `ENV` doesn't perform any substitution which is the reason for this error: https://github.com/facebookincubator/velox/actions/runs/10105042990/job/27945816369?pr=10446#step:5:137 so you'll have to hard code that value

Reply:
@assignUser Yes. Removed the java install command. And also update the hadoop classpath setting code.